taskchampion/src/taskdb/mod.rs

```diff
@@ -31,6 +34,54 @@ impl TaskDb {
         TaskDb::new(Box::new(InMemoryStorage::new()))
     }
 
+    /// Apply `operations` to the dataabase, in a single transaction.
+    ///
+    /// The resulting operations will be appended to the list of local operations, and the
```

Comment:
```suggestion
    /// The operations will be appended to the list of local operations, and the
```

It's all the operations passed into the function, right?